tests/Doctrine/Tests/Common/Collections/BaseCollectionTest.php

```diff
@@ -249,5 +249,6 @@ public function testMatchingAlwaysReturnsCollection(): void
         $criteria = Criteria::create();
 
         self::assertInstanceOf(Collection::class, $this->collection->matching($criteria));
+        self::assertInstanceOf(Selectable::class, $this->collection->matching($criteria));
```

Comment:
We don't do semantic commits here but anyway, why did you prefix this one with `bugfix:` ? It only modifies a test…

Reply:
it was a bug in the test tho. I did not assert what I wanted to assert. Feel free to squash this PR tho.

Reply:
Squashed my commits now.